carddrawer/build.gradle

```diff
@@ -24,6 +24,7 @@ android {
     testOptions {
         unitTests {
             includeAndroidResources = true
+            unitTests.returnDefaultValues = true
```

Comment:
I don't think you need the **unitTests** because it's already inside and a **unitTests**.

Reply:
Adjustment completed